File: app/src/main/java/info/firozansari/animatedvectordrawable/AnimatedIcon.java
package info.firozansari.animatedvectordrawable;

import android.graphics.drawable.AnimatedVectorDrawable;
import android.support.graphics.drawable.AnimatedVectorDrawableCompat;

/**
 * Created by firoz on 25/03/2017.
 */

class AnimatedIcon {

    public AnimatedVectorDrawableCompat icon;

    public String title;

    public AnimatedIcon(AnimatedVectorDrawableCompat icon, String title) {

        this.icon = icon;
        this.title = title;
    }

    public AnimatedVectorDrawableCompat getIcon() {
        return icon;
    }

    public void setIcon(AnimatedVectorDrawableCompat icon) {
        this.icon = icon;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }
}

File: app/src/main/java/info/firozansari/animatedvectordrawable/AnimatedIcon.kt
package info.firozansari.animatedvectordrawable

import androidx.vectordrawable.graphics.drawable.AnimatedVectorDrawableCompat

/**
 * Created by firoz on 25/03/2017.
 */
internal class AnimatedIcon(icon: AnimatedVectorDrawableCompat, title: String) {
    var icon: AnimatedVectorDrawableCompat
    var title: String

    init {
        this.icon = icon
        this.title = title
    }

    fun getIcon(): AnimatedVectorDrawableCompat {
        return icon
    }

    fun setIcon(icon: AnimatedVectorDrawableCompat) {
        this.icon = icon
    }
}